Tasks/google-play-release/task.json

```diff
@@ -93,23 +93,32 @@
             "label": "Release Notes (File)",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "Path to the file specifying the release notes for the APK you are publishing."
+            "helpMarkDown": "Path to the file specifying the release notes (change log) for the APK you are publishing."
         },
         {
             "name": "shouldAttachMetadata",
             "type": "boolean",
             "label": "Update Metadata",
             "defaultValue": false,
             "required": false,
-            "helpMarkDown": "Checking this box will make the task attempt to update the metadata on your app release"
+            "helpMarkDown": "Checking this box will make the task attempt to update the metadata on your app release."
         },
         {
             "name": "metadataRootPath",
             "type": "filePath",
             "label": "Metadata Root Directory",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "The path to the metadata folder with the fastlane metadata structure",
+            "helpMarkDown": "The path to the metadata folder with the fastlane metadata structure.",
+            "visibleRule": "shouldAttachMetadata = true"
+        },
+        {
+            "name": "shouldUploadApks",
+            "type": "boolean",
+            "label": "Update APK(s)",
+            "defaultValue": true,
+            "required": false,
+            "helpMarkDown": "By default, the task will update the specified binary APK file(s) on your app release. Unselecting this option you can update metadata keeping the APKs untouched.",
```

Comment:
Suggestion:
Change:
"Unselecting this option you can"
To:
"By unselecting this option, you can"